src/devices/Bmxx80/Bmxx80Base.cs

```diff
@@ -121,14 +121,16 @@ protected virtual Temperature CompensateTemperature(int adcTemperature)
             // See: https://cdn-shop.adafruit.com/datasheets/BST-BMP280-DS001-11.pdf
             double var1 = ((adcTemperature / 16384.0) - (_calibrationData.DigT1 / 1024.0)) * _calibrationData.DigT2;
             double var2 = (adcTemperature / 131072.0) - (_calibrationData.DigT1 / 8192.0);
-            var2 *= var2 * _calibrationData.DigT3;
+            var2 *= var2 * _calibrationData.DigT3 * GetTempCalibrationFactor();
 
             TemperatureFine = (int)(var1 + var2);
 
             double temp = (var1 + var2) / 5120.0;
             return Temperature.FromCelsius(temp);
         }
 
+        protected virtual int GetTempCalibrationFactor() => 1;
```

Comment:
NIT: doesn't have to be a method if you don't want to, you can just do a property getter so that you use it as if it were a field like:
```c#
protected override int _factor => 1;
```

It's up to you, whatever you think is cleanest is fine. I just mention this because I saw that you commented on if this could just be a field, so using virtual properties would be the closest thing.

Reply:
Yeah I was thinking of a virtual property I'll change that.